modules/lang-painless/src/main/java/org/elasticsearch/painless/DefBootstrap.java

```diff
@@ -445,7 +445,7 @@ public static CallSite bootstrap(PainlessLookup painlessLookup, FunctionTable fu
         switch(flavor) {
             // "function-call" like things get a polymorphic cache
             case METHOD_CALL:
-                if (args.length == 0) {
+                if (args.length < 2) {
```

Comment:
Do we always pass in the script as an argument? Seems like we should only do that when it's actually required?

Reply:
We pass in two things indy - the recipe like before and whether or not we passed in the script. arg[1] 0 if we didn't pass the script and 1 if we did.